### PR TITLE
docs: Use backticks for commands and code words

### PR DESCRIPTION
**Pull request by ahawkins**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2752)

### CODE REVIEW

File: docs/cli/kops_rolling-update_cluster.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ Rolling update a cluster.
...
 
-To perform rolling update, you need to update the cloud resources first with "kops update cluster" 
+To perform rolling update, you need to update the cloud resources first with `kops update cluster` 
 
-Note: terraform users will need run the following commands all from the same directory "kops update cluster --target=terraform" then "terraform plan" then "terraform apply" prior to running "kops rolling-update cluster" 
+Note: terraform users will need run the following commands all from the same directory `kops update cluster --target=terraform` then `terraform plan` then `terraform apply` prior to running `kops rolling-update cluster` 
 
-Use export KOPS FEATURE FLAGS="+DrainAndValidateRollingUpdate" to use beta code that drains the nodes and validates the cluster.  New flags for Drain and Validation operations will be shown when the environment variable is set.
+Use export `KOPS FEATURE FLAGS="+DrainAndValidateRollingUpdate"` to use beta code that drains the nodes and validates the cluster.  New flags for Drain and Validation operations will be shown when the environment variable is set.
```
Comment (chrislovecnm) [inline docs/cli/kops_rolling-update_cluster.md]: do we want to have "export" back ticked as well?

**Comment by ahawkins** [comment]
yup!


On Fri, Jun 16, 2017, at 12:08 AM, Chris Love wrote:
> *@chrislovecnm* commented on this pull request.


> Question for you


> 
> In docs/cli/kops_rolling-update_cluster.md[1]:


> > -Use export KOPS FEATURE FLAGS="+DrainAndValidateRollingUpdate" to
> > use beta code that drains the nodes and validates the cluster.  New
> > flags for Drain and Validation operations will be shown when the
> > environment variable is set. +Use export `KOPS FEATURE
> > FLAGS="+DrainAndValidateRollingUpdate"` to use beta code that
> > drains the nodes and validates the cluster.  New flags for Drain
> > and Validation operations will be shown when the environment
> > variable is set.
>> do we want to have "export" back ticked as well?


> — You are receiving this because you were mentioned. Reply to this
> email directly, view it on GitHub[2], or mute the thread[3].> 


Links:

  1. https://github.com/kubernetes/kops/pull/2752#discussion_r122279787
  2. https://github.com/kubernetes/kops/pull/2752#pullrequestreview-44382746

**Comment by chrislovecnm** [comment]
Completely missed this, my apologies.  This document cannot be directly edited, as it is autogenerated by go code.  You need to edit the documentation here: https://github.com/kubernetes/kops/blob/master/cmd/kops/rollingupdatecluster.go#L44 and then run `make gen-cli-docs`.

**Comment by chrislovecnm** [comment]
@k8s-bot ok to test

**Comment by justinsb** [comment]
Check out #2943... it turned out there were a couple of gotchas.  But once that merges, this should be possible :-)

**Pull request closed without merging**